gen.py

```diff
@@ -72,15 +72,25 @@ class PlotFlags(OutFlags):
     template: FilePath = Field(None, description="Default plot template")
 
 
+# pylint: disable=invalid-name
+class X(BaseModel):
+    __root__: dict[FilePath, PlotColumn] = Field(None)
+
+
+# pylint: disable=invalid-name
+class Y(BaseModel):
+    __root__: dict[FilePath, PlotColumn | PlotColumns] = Field(None)
```

Comment:
I know the types are confusing, but `PlotColumns` is union of `PlotColumn | set[PlotColumn]`, so we don't need a union here.
```suggestion
    __root__: dict[FilePath, PlotColumns] = Field(None)
```